### PR TITLE
oyunbileg-davaanyam

### PR DESCRIPTION
**Pull request by Oyunbileg-dav**

### CODE REVIEW

File: 2021/fall/Sterne-TTh@10/oyunbileg-davaanyam.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Hi I'm Oyu!
```
Comment (yutaro-shimizu) [inline 2021/fall/Sterne-TTh@10/oyunbileg-davaanyam.md]: This file seems good to me. I'm not sure about the second file though, it seems you pushed an empty file under the fall directory.

**Review by yutaro-shimizu — COMMENTED** [review]
This file seems good to me. I'm not sure about the second file though, it seems you pushed an empty file under the fall directory.

File: 2021/fall/Sterne-TTh@10/oyunbileg-davaanyam.md (hunk 1 above)
Comment (Oyunbileg-dav) [reply 2021/fall/Sterne-TTh@10/oyunbileg-davaanyam.md]: Thanks for the comment! Deleted the empty file :)

**Review by Bojne — APPROVED** [review]
LGTM!

**Pull request merged**